DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -79,6 +79,7 @@ public void ConfigureServices(IServiceCollection services)
                 .AddRazorOptions(options =>
                 {
                     options.ViewLocationFormats.Add("/Views/TrackingSystem/{1}/{0}.cshtml");

```

Comment:
Can we remove this option, as you mentioned earlier? :)

Reply:
We can't - it breaks one of the other controllers. I assumed that was being changed in the other branch.

Reply:
Ah, fair enough! Thanks :)